tests/fixtures/mda.yaml

```diff
@@ -13,16 +13,35 @@ grid_plan:
 metadata:
   some info: something
 stage_positions:
-- x: 10.0
+- name: null
+  sequence: null
+  x: 10.0
   y: 20.0
+  z: null
 - name: test_name
+  sequence:
```

Comment:
for the `mda.yaml` fixture, leave out all the fields that are just their default values (like `channels: []` or `go_up: true`, etc...).  they have to be in the json fixture, but not this one